addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -1158,13 +1158,14 @@ var MediaDialog = Dialog.extend({
 
         this.opened(function () {
             var tabToShow = 'icon';
+            var parentClassAttr = self.$media.parent().attr('class');
             if (!self.media || self.$media.is('img')) {
                 tabToShow = 'image';
             } else if (self.$media.is('a.o_image')) {
                 tabToShow = 'document';
             } else if (self.$media.attr('class').match(/(^|\s)media_iframe_video($|\s)/)) {
```

Comment:
And what about this one ?

Reply:
This one wasn't buggy my side

Reply:
... because you have a class in your case but this is the exact same potential bug ...